site/sidebars.js

```diff
@@ -412,6 +412,10 @@ const sidebars = {
       type: 'doc',
       id: 'releases',
     },
+    {
```

Comment:
I'm not sure if this should take up real estate above enterprise/red team docs. Open to thoughts here